### PR TITLE
14810-datasets-names-in-ckan-are-misleading

### PR DESCRIPTION
**Pull request by lgudyma-kontur**

### CODE REVIEW

File: scripts/ma_dictionaries.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "bdi": "Burundi",  # Country
     "bel": "Belgium",  # Country
     "ben": "Benin",  # Country
-    "bes": "Bonaire, Sint Eustatius and Saba",  # Country
+    "bes": "Bonaire, (Sint Eustatius and Saba)",  # Country
```
Comment (Zkir) [inline scripts/ma_dictionaries.py:36]: In this case parentheses are not necessary. 
"Bonaire" is not a shorter name of a country.

File: scripts/ma_dictionaries.py [hunk 2]
```diff
@@ -209,7 +209,7 @@
     "sen": "Senegal",  # Country
     "sgp": "Singapore",  # Country
     "sgs": "South Georgia and the South Sandwich Islands",  # Country
-    "shn": "Saint Helena, Ascension and Tristan da Cunha",  # Country
+    "shn": "Saint Helena, (Ascension and Tristan da Cunha)",  # Country
```
Comment (Zkir) [inline scripts/ma_dictionaries.py:212]: Should be `Saint Helena, Ascension and Tristan da Cunha`

**Review by Zkir — CHANGES_REQUESTED** [review]

**Comment by Zkir** [comment]
@lgudyma-kontur please change according to comments.

**Review by Zkir — APPROVED** [review]
looks ok

File: scripts/ma_dictionaries.py [hunk 3]
```diff
@@ -90,7 +90,7 @@
     "flk": "Falkland Islands (Malvinas)",  # Country
     "fra": "France",  # Country
     "fro": "Faroe Islands",  # Country
-    "fsm": "Micronesia, Federated States of",  # Country
+    "fsm": "Micronesia (Federated States of)",  # Country
```
Comment (amurKontur) [inline scripts/ma_dictionaries.py:93]: I don't think that this fix is OK supported in future if other datasets are added. But as for now we can leave it as it is

**Pull request merged**